brigade/static/scss/molecules/_alert.scss

```diff
@@ -0,0 +1,18 @@
+@import '../core/variables';
+
+//
+// Alert
+//
+
+.alert-success, 
+.alert-failure, 
+.alert-pending, 
+.alert-None {
```

Comment:
lol, I see that you just copied this from the existing code, but... `.alert-None` seems like we are trying to style a python bug.